src/pages/Run.tsx

```diff
@@ -49,43 +49,45 @@ export default ({ runId }: { runId: string }) => {
     gql`
       query SINGLE_RUN_QUERY($runId: String!) {
         runs(filter: { runId: $runId }) {
-          runId
-          sessionId
-          commandLine
-          completeTime
-          duration
-          engineParameters {
-            launchDir
-            projectDir
-            resume
-            revision
-            workDir
-          }
-          errorReport
-          exitStatus
-          parameters
-          repository
-          startTime
-          state
-          success
-          tasks {
-            taskId
+          content {
```

Comment:
what the, not sure why git makes it look like this in my vscode it looks like this
![image](https://user-images.githubusercontent.com/34698315/104511454-bbebc180-55ba-11eb-980e-ce129e35ba73.png)